src/frontend/typer/behandling.ts

```diff
@@ -27,6 +27,7 @@ export enum BehandlingÅrsak {
     ÅRLIG_KONTROLL = 'ÅRLIG_KONTROLL',
     DØDSFALL = 'DØDSFALL',
     NYE_OPPLYSNINGER = 'NYE_OPPLYSNINGER',
+    KLAGE = 'KLAGE',
```

Comment:
Skal man alltid få lov til å opprette klager? Vi har en del logikk på opprettelse av revurdering vet jeg så vil tro at noe av den samme logikken ligger til grunn for klage :)

Reply:
Etter diskusjon med Eivind så er det sånn at i første omgangen så blir det alltid lov å opprette klage ved revurdering.